spec/menulist_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'menulist'
+describe MenuList do
+  let (:including_class){ Class.new { include MenuList } }
+
+  describe '#select'do
+    it 'allows you to select a menu'do
+      expect(including_class.new.select(:burger)).to eq ({:Hamburger => 4.50, :Cheeseburger => 4.60, :Chips => 1.00} )
+    end
+    it 'allows you to select a menu'do
+      expect(including_class.new.select(:wine)).to eq ({:red => 3.00, :white => 3.00, :blue => 10.00})
```

Comment:
Lint/ParenthesesAsGroupedExpression: `(...)` interpreted as grouped expression. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)<br>Metrics/LineLength: Line is too long. [102/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)<br>Style/RedundantParentheses: Don't use parentheses around a literal.